mitodl/hugo-course-publisher
# update course info sidebar, overall course layout

#### Pre-Flight checklist


#### What are the relevant tickets?

https://trello.com/c/aOlRClVV/2-redesign-ocw-course

#### What's this PR do?

a few things, related to getting the 'course info' bar in shape.

- I redid the overall layout to have even spacing around the left-side nav, the main content (the middle), and the course info panel
- I added an `.active` class on the link which is currently active in the desktop nav and styled that so you can tell what page you're on
- re-style the course info panel in line with the designs

I also did some refactoring to reduce duplication between the `site/layout/courses/section.html` and `site/layout/courses/single.html` files. before they were both defining the columns and so on, which meant that changing the design required editing both files in the same way.

what I did to fix that was to move the shared code out into `site/layout/courses/_default.html` so that the other two files only have the markup in them which is specific to them. 

Note that this doesn't deal with the mobile UI yet! I thought this was a big enough PR already.

#### How should this be manually tested?

check out some course pages and make sure things are working alright. there should be a 'coming soon' popup on a lot of the links when you click on them.

#### Screenshots (if appropriate)

![Screenshot from 2020-02-06 11-32-42](https://user-images.githubusercontent.com/6207644/73957478-68fe8a00-48d4-11ea-87c2-6bba47d7e15b.png)
![Screenshot from 2020-02-06 11-32-37](https://user-images.githubusercontent.com/6207644/73957479-69972080-48d4-11ea-91b4-a60781de2177.png)
![Screenshot from 2020-02-06 11-32-30](https://user-images.githubusercontent.com/6207644/73957482-6a2fb700-48d4-11ea-85f1-b23cf15f54a4.png)
